codefordc/housing-insights
# 505 base proj apiconn

## What's In This
Generate _project.csv and _addre.csv according to these rules:
- don't add a new row if there is a duplicate project with different unique_data_id
- add new row if there is a duplicate project with same unique_data_id
- add a new row if no matching project is found

Uses HILogger to log results from determining whether to add new row or
not. Merged in 136-fix-dhcd-loan-table-download branch. (#570) 

## Testing
- run get_api_data.py on "DCHousing" and "dhcd" modules and verify that appropriate csv files are generated. The resulting files will be unique projects for the respective raw data vs prescat projects.
- update manifest accordingly, load dchousing_addre into proj_addre, rename current _addre.csv files, and then re-run get_api_data.py on "DCHousing" and "dhcd". The resulting dchousing_addre.csv should be same as the original and "dhcd" should be shorter to account for duplicates across dchousing and dhcd.
